Tools/Frame_params/ModalAI/AutonomyDevKit.parm

```diff
@@ -0,0 +1,80 @@
+# parameters for the autonomy dev kit
```

Comment:
MP's parameter loader apparently can't see files below the "Frame_params" directory so perhaps we should move this up and add a prefix.  E.g. "ModalAI_AutonomyDevKit.parm"

Reply:
as discussed on the call, once merged I will do a MissionPlanner PR to support this directory, like I did for quadplanes